cmd/limactl/main.go

```diff
@@ -92,9 +93,23 @@ func newApp() *cobra.Command {
 		}
 		// Make sure either $HOME or $LIMA_HOME is defined, so we don't need
 		// to check for errors later
-		if _, err := dirnames.LimaDir(); err != nil {
+		dir, err := dirnames.LimaDir()
+		if err != nil {
 			return err
 		}
+		// Make sure that directory is on a local filesystem, not on NFS
+		// if the directory does not yet exist, check the home directory
+		_, err = os.Stat(dir)
+		if errors.Is(err, os.ErrNotExist) {
+			dir = filepath.Dir(dir)
+		}
+		nfs, err := fsutil.IsNFS(dir)
+		if err != nil {
+			return err
+		}
+		if nfs {
+			return errors.New("must not run on NFS dir")
```

Comment:
Probably this should be just a warning?

It still may work on NFS with a future potential version of QEMU, vz, or something else?

Reply:
Possibly, but I'm not sure if it is something that can be supported?

We can download/copy the "basedisk" from a remote location, such as ~/.cache, but keeping the "diffdisk" on a network file share could be a potential source of bugs and performance issues. Even if the bug here, regarding file locking with `qemu-img`, can be found and fixed? Something like iSCSI could work, but skeptical about NFS.